docs/context_menu.md

```diff
@@ -33,20 +38,47 @@ class zen_context_menu_actions {
 
 ## Adding Actions Through Script
 
-Context menu actions can be added by calling the `zen_context_menu_fnc_addAction` function.
+#### Creating an Action
+
+Context menu actions can be added by first creating an action with the `zen_context_menu_fnc_createAction` function.
+This function is used to ensure that the created action array is in the correct format.
+
+**Arguments:**
+
+ \#   | Description | Type | Default Value (if optional)
+:---: | ----------- | ---- | ---------------------------
+0 | Action Name | STRING |
+1 | Display Name | STRING |
+2 | Icon and Icon Color | STRING or ARRAY | ["", [1, 1, 1, 1]]
+3 | Statement | CODE |
+4 | Condition | CODE | {true}
+5 | Arguments | ANY | []
+6 | Dynamic Children | CODE | {}
+7 | Modifier Function | CODE | {}
+
+**Return Value:**
+
+&nbsp;&nbsp;&nbsp;&nbsp;Action &lt;ARRAY&gt;
+
+**Example:**
+
+```clike
```

Comment:
Doesn't Jekyll support `sqf`?

Reply:
The docs don't use jekyll.

Reply:
Whoops, but still, doesn't the current docs support GitHub flavoured markdown?

Reply:
No, unfortunately it doesn't. It uses prism for the syntax highlighting.